front/src/services/aleph.ts

```diff
@@ -119,4 +116,56 @@ export class AlephService {
 	async deleteFiles(itemHashes: string[]): Promise<ForgetMessage> {
 		return this.subAccountClient.forget({ hashes: itemHashes });
 	}
+
+	async createAggregate<T extends Record<string, unknown>>(key: string, content: T): Promise<AggregateMessage<T>> {
+		return this.subAccountClient.createAggregate({
+			key,
+			content,
+			channel: ALEPH_GENERAL_CHANNEL,
+			address: this.account.address,
+		});
+	}
+
+	async fetchAggregate<T extends z.ZodTypeAny>(key: string, schema: T) {
+		const { success, data, error } = schema.safeParse(
+			await this.subAccountClient.fetchAggregate(this.account.address, key).catch(() => {}),
```

Comment:
not sure about the silenced catch but ok, we'll see later if we have an issue

Reply:
The catch is there to default if there is no value (HTTP 404) there will be a problem if the error is something else but I guess it's not that important atm